doc/DeveloperGuide.md

```diff
@@ -87,6 +87,39 @@ Use case ends.
 
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
+  
+### Use case: renaming an existing tag
+
+**MSS**
+
+1. User requests to rename an existing tag
```

Comment:
this one should be something like `user list existing tags`

Reply:
my idea is that user input command like "change tag" and the app will list out all the existing tags used in step 2.